index.html

```diff
@@ -5099,6 +5125,31 @@ <h2 id="defining-a-valid-audioworkletprcoessor">
                 <code>"name"</code>.
               </li>
             </ol>
+            <p>
+              The return value of this method controls the lifetime of the
+              <a>AudioWorkletProcessor</a>'s associated <a>AudioWorkletNode</a>
+              by determining the existence of the node's <a href=
+              "#dfn-active-reference">active reference</a>. At the conclusion
+              of each call to the <code>process()</code> method, if the result
+              of applying <a href=
+              "https://tc39.github.io/ecma262/#sec-toboolean"><code>ToBoolean</code></a>
+              (described in [[!ECMASCRIPT]]) to the return value is equal to
+              <code>true</code>, then a single <a>active reference</a> is
+              retained to the node, affecting the node's <a href=
+              "#lifetime-AudioNode">lifetime</a>. Otherwise, any active
+              reference is removed and the node's lifetime depends only on
+              other references as they may exist.
+            </p>
+            <div class="note">
+              While returning <code>true</code> ensures that
+              <code>process()</code> will be called during subsequent rendering
+              quanta, returning <code>false</code> does not prevent future
+              calls to <code>process()</code>. Other references to the
+              corresponding <a>AudioWorkletNode</a> may exist and the node may
+              thus continue to function. Furthermore, these future calls may
+              return a value of <code>true</code>, reinstating the active
+              reference.
```

Comment:
Let me clarify my idea first:

These two conditions must be met first:
- The JS reference to an AWN is dropped.
- There is no active input to an AWP.

__Then__ UA starts checking the return value of process(). Note that all previous return values must be ignored because the lifetime is cannot be inactive.

When `AWP::process()` returns `false`, UA put this instance of AWN/AWP into the GC queue. Eventually it will be collected.

Based on this algorithm, you cannot reinstate the processor by returning `true` again. When UA starts checking the return value of `process()` method, your JS reference is already dropped. Thus you cannot call `.connect()` method on it.